docs/src/visualization.md

```diff
@@ -174,7 +174,7 @@ julia> compute_vorticity(velocity, semi) =
          compute_vorticity(velocity, Trixi.mesh_equations_solver_cache(semi)...);
 
 julia> function get_velocity(sol)
-         rho, rhou, rhov, E = StructArrays.components(sol.u[end])
+         rho, rhou, rhov, E = StructArrays.components(Base.parent(sol.u[end]))
```

Comment:
This is a breaking PR since we changed documented behavior. I will mark it accordingly and take care of it.